eng/common/scripts/Package-Properties.ps1

```diff
@@ -91,7 +91,7 @@ function Get-PkgProperties
     {
         $pkgDirectoryPath = Join-Path $serviceDirectoryPath $directory.Name
 
-        if ((Get-ChildItem -Path Function: | ? { $_.Name -eq $GetPackageInfoFromRepoFn }).Count -gt 0)
+        if (Test-Path Function:GetPackageInfoFromRepoFn)
```

Comment:
```suggestion
        if (Test-Path "Function:$GetPackageInfoFromRepoFn")
```

Reply:
This seems to always return true. This won't work because its actually a variable. I'll leave the previous logic.